custom-connectors/NSF Data/apiProperties.json

```diff
@@ -0,0 +1,58 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "host": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Host (with https/http)",
+          "description": "Your Domino server URL",
+          "tooltip": "Provide the Domino host URL with scheme https/http",
+          "constraints": {
+            "tabIndex": 1,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "username": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Username",
+          "description": "The username for this api",
+          "tooltip": "Provide the username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Password",
+          "description": "The password for this api",
+          "tooltip": "Provide the password",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Dynamic host",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('host')"
+        }
+      }
+    ],
+    "publisher": "Databoat AG",
+    "stackOwner": "HCL Technologies"
```

Comment:
@DataboatDev Can you please let us know the relationship b/w `Databoat AG` and `HCL Technologies`. If the connector is being published by `Databoat AG`, the `stackOwner` should be the same.

Reply:
@sriyen-msft This is a generic Connector that allows communication with HCL Domino on-premise installations. Due to NDA I cannot provide more details here. Can we take it offline? Perhaps you can find more details in your CRM. Communications were done with a colleague of you who unfortunately just left your team.